steam_wrapper/steam_wrapper.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+import os
+import os.path
+import sys
+import shutil
+import errno
+import fnmatch
+import subprocess
+
+STEAM_PATH = "/app/bin/steam"
+STEAM_ROOT = os.path.expandvars("$HOME/.var/app/com.valvesoftware.Steam")
+
+def prompt():
+    p = subprocess.Popen(["zenity", "--question",
+                          ("--text="
+                           "This is com.valvesoftware.Steam cloud sync repair. "
+                           "If you have conflicting local and cloud data for "
+                           "your game, this may result in partial loss of your "
+                           "cloud data. If you instead prefer ensuring cloud data "
+                           "persists, please relocate your "
+                           "~/.var/app/com.valvesoftware.Steam/data/Steam "
+                           "to a secure location, "
+                           "remove ~/.var/app/com.valvesoftware.Steam "
+                           "and put Steam data directory back to avoid needing to "
+                           "re-download games")])
```

Comment:
Small typo: Missing dot at the end of the sentence. ;-) Also, the window has an empty (default) icon which looks bad. Would it be possible to replace it with a question icon?